### PR TITLE
Released v22.1.34

### PR DESCRIPTION
**Pull request by SathishKumarRajendran**
## Feature Description

Added the new locale text for v22.1.34

### CODE REVIEW

**Review by Mydeensn — APPROVED** [review]
Changes are fine

File: src/is.json [hunk 1]
```diff
@@ -507,7 +509,7 @@
       "FormatCellsThatContainTheText": "Snið hólf sem innihalda textann:",
       "FormatCellsThatContainADateOccurring": "Snið hólf sem innihalda dagsetningu sem á sér stað:",
       "FormatCellsDuplicate": "Forsníða frumur sem innihalda:",
-      "FormatCellsTop": "Forsníða frumur sem raðast efst:",
```
Comment (PiramanayagamSyncfusion) [inline src/is.json:510]: revert this changes

File: src/is.json [hunk 2]
```diff
@@ -1068,7 +1071,7 @@
       "cut": "Skera",
       "copy": "Afrita",
       "paste": "Líma",
-      "unorderedList": "Punktalista",
```
Comment (PiramanayagamSyncfusion) [inline src/is.json:1071]: revert this

File: src/is.json [hunk 3]
```diff
@@ -1865,7 +1919,7 @@
       "Table": "Tafla",
       "Row": "Röð",
       "Cell": "Cell",
-      "Ok": "OK",
```
Comment (PiramanayagamSyncfusion) [inline src/is.json:1868]: revert this

File: src/is.json [hunk 4]
```diff
@@ -300,6 +300,8 @@
       "DeleteColumn": "Eyða dálki",
```
Comment (PiramanayagamSyncfusion) [inline src/is.json:300]: revert this entire file changes and add specific files changes

**Review by PiramanayagamSyncfusion — APPROVED** [review]
changes fine
